renderdoc/data/glsl/mesh.comp

```diff
@@ -40,37 +40,87 @@ layout(binding = 3, std140) buffer pickresult_buffer
 
 layout (local_size_x = 128, local_size_y = 1, local_size_z = 1) in;
 
+bool TriangleRayIntersect(vec3 A, vec3 B, vec3 C, 
+	vec3 RayPosition, vec3 RayDirection, out vec3 HitPosition)
+{
+	bool Result = false;
+
+	vec3 v0v1 = B - A;
+	vec3 v0v2 = C - A;
+	vec3 pvec = cross(RayDirection, v0v2);
+	float det = dot(v0v1, pvec);
+
+	// if the determinant is negative the triangle is backfacing
+	// if the determinant is close to 0, the ray misses the triangle
+	if (abs(det) > 0.000001f)
```

Comment:
Yea I'd like it to pick on backfaces too. It does currently, but it's acting a little funny so I need to do some debugging. That note and code is kind of misleading. I left it there to remind myself how to add backface culling if I wanted it (don't take abs(det) on line 55). I had copy/pasted this function from my personal collision detection library.